roles/tower-setup/tasks/main.yaml

```diff
@@ -122,7 +121,7 @@
   no_log: yes
   vars:
     license: "{{ tower_setup_license | from_json }}"
-  command: tower-cli setting modify LICENSE '{{ license | to_json }}'
+  command: tower-cli setting modify -u '{{ tower_admin_username }}' -p '{{ tower_admin_password }}' LICENSE '{{ license | to_json }}'
```

Comment:
This might be an issue because the variables are defined in the tower-config role and this is in the tower-setup role.

Reply:
Nevermind, saw that you added the variables in the role